src/Android/Xamarin.Android/Samples/Symbology/UseDistanceCompositeSym/UseDistanceCompositeSym.cs

```diff
@@ -0,0 +1,113 @@
+// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
+using Android.App;
+using Android.OS;
+using Android.Widget;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
+
+namespace ArcGISRuntimeXamarin.Samples.UseDistanceCompositeSym
+{
+    [Activity]
+    public class UseDistanceCompositeSym : Activity
+    {
+        // Create and hold reference to the used MapView
+        private SceneView _mySceneView = new SceneView();
+
+        // URL for an image service to use as an elevation source
+        private string _elevationSourceUrl = @"http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer";
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+
+            Title = "Use distance composite symbol";
+
+            // Create the UI
+            CreateLayout();
+
+            // Create the Scene, basemap, graphic, and composite symbol
+            Initialize();
+        }
+
+        private void Initialize()
+        {
+            // Create a new Scene with an imagery basemap
+            Scene myScene = new Scene(Basemap.CreateImagery());
+
+            // Create an elevation source for the Scene
+            ArcGISTiledElevationSource elevationSrc = new ArcGISTiledElevationSource(new Uri(_elevationSourceUrl));
+            myScene.BaseSurface.ElevationSources.Add(elevationSrc);
+
+            // Add the Scene to the SceneView
+            _mySceneView.Scene = myScene;
+
+            // Create a new GraphicsOverlay and add it to the SceneView
+            GraphicsOverlay grafixOverlay = new GraphicsOverlay();
+            grafixOverlay.SceneProperties.SurfacePlacement = SurfacePlacement.Relative;
+            _mySceneView.GraphicsOverlays.Add(grafixOverlay);
+
+            // Call a function to create a new distance composite symbol with three ranges
+            DistanceCompositeSceneSymbol compositeSymbol = CreateCompositeSymbol();
+
+            // Create a new point graphic with the composite symbol, add it to the graphics overlay
+            MapPoint heliPoint = new MapPoint(-2.708471, 56.096575, 5000, SpatialReferences.Wgs84);
```

Comment:
Worth while changing the name of this var now that we aren't using the the helicopter symbol